todo.sh

```diff
@@ -575,21 +575,24 @@ do
             export HIDE_PRIORITY_SUBSTITUTION="([A-Z])[[:space:]]"
         fi
         ;;
-    t )
+    t)
         OVR_TODOTXT_DATE_ON_ADD=1
         ;;
-    T )
+    T)
         OVR_TODOTXT_DATE_ON_ADD=0
         ;;
-    v )
+    v)
         : $(( TODOTXT_VERBOSE++ ))
         ;;
-    V )
+    V)
         version
         ;;
-    x )
+    x)
         OVR_TODOTXT_DISABLE_FILTER=1
         ;;
+    *) 
+        : $(( TODOTXT_VERBOSE++ ))
```

Comment:
It would be better to fail (i.e. call usage()) on unknown options than to increase verbosity.